Makefile

```diff
@@ -10,7 +10,7 @@ GIT_BRANCH := $(shell git branch | grep \* | cut -d ' ' -f2)
 GIT_HASH := $(GIT_BRANCH)/$(shell git log -1 --pretty=format:"%H")
 TIMESTAMP := $(shell date '+%Y-%m-%d_%I:%M:%S%p')
 CONTROLLER_GEN=controller-gen
-CONTROLLER_GEN_REQ_VERSION := v0.8.0
+CONTROLLER_GEN_REQ_VERSION := v0.9.1-0.20220629131006-1878064c4cdf
```

Comment:
Looks like changes has not included in latest release 0.9.2 ?

Reply:
Nope, not yet :(

Reply:
@prateekpandey14 I need to cherry pick my changes so that they are included in the next release.
Until then it's still better to use the official repo than my fork IMHO.